### ISSUE TITLE
MSELoss

### ISSUE THREAD

**Issue by nagataka** [nagataka/PyTorch-Code-Reading#1]
Simple and familiar thing first...

#### [pytorch/torch/nn/modules/loss.py (class: MSELoss)](https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/loss.py#L368)
Simply call ```F.mse_loss(input, target, reduction=self.reduction)``` and return it.

#### [pytorch/torch/nn/functional.py(function: mse_loss)](https://github.com/pytorch/pytorch/blob/master/torch/nn/functional.py#L2186)

First, compute squared error ```ret = (input - target) ** 2```.
Then, perform reduction based on parameter: 'mean' or 'sum' (default: 'mean') ```ret = torch.mean(ret) if reduction == 'mean' else torch.sum(ret)``` where reduction is defined in [pytorch/torch/nn/_reduction.py](https://github.com/pytorch/pytorch/blob/master/torch/nn/_reduction.py) which simply returns 0(none), 1(mean), or 2(sum)
